client/js/app/main/modules/week/week.config.js

```diff
@@ -4,9 +4,11 @@
 module.exports = function ($stateProvider) {
     $stateProvider.state('main.week', {
         url: '/week',
+        controller: require('./week.controller'),
+        controllerAs: 'week',
         templateUrl: {
```

Comment:
Make $templateCache object and don't use root: __dirname for template. This issue connected to all templates.

For resolve it we need to create gulp tusk for it.